### DISCUSSION TITLE
i cant open my github.io page

### DISCUSSION THREAD

**Discussion by winksolutions** [community/community#32652] [Other Feature Feedback, Questions, & Ideas] [answered]
[winksolutions.github.io/mrrobot](https://winksolutions.github.io/mrrobot) **ahows error 404**

**Comment by airtower-luna** [comment, marked as answer]
There doesn't seem to be any build/deployment, did you [enable Pages for the repository](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-your-site)?

**Reply by winksolutions** [reply]
thanks everything now works fine!

**Reply by airtower-luna** [reply]
Excellent! Please don't forget to mark the answer. :slightly_smiling_face:

**Comment by nazmgm** [comment]
i cant open pages for repository no option for me how can i fix it
